lib/tapioca/gemfile.rb

```diff
@@ -53,21 +53,29 @@ def require
 
     sig { returns([T::Array[Gem], T::Array[String]]) }
     def load_dependencies
-      deps = definition.locked_gems.dependencies.values
-
-      missing_specs = T::Array[String].new
-
-      dependencies = definition
-        .resolve
-        .materialize(deps, missing_specs)
+      materialized_dependencies, missing_specs = materialize_deps
+      dependencies = materialized_dependencies
         .map { |spec| Gem.new(spec) }
         .reject { |gem| gem.ignore?(dir) }
         .uniq(&:rbi_file_name)
         .sort_by(&:rbi_file_name)
-
       [dependencies, missing_specs]
     end
 
+    sig { returns([T::Array[::Gem::Specification], T::Array[String]]) }
+    def materialize_deps
+      deps = definition.locked_gems.dependencies.values
+      missing_specs = T::Array[String].new
+      materialized_dependencies = T::Array[Gem].new
+      if ::Gem.loaded_specs["bundler"].version < ::Gem::Version.create("2.2.25")
```

Comment:
Can we just check for the arity of the method instead of the Bundler version, please?

Reply:
Ah sure, good call.